### ISSUE TITLE
As a developer, I'd like better handling around when the config file is missing.

### ISSUE THREAD

**Issue by adigitalmonk** [adigitalmonk/durendal-bot#5]
As a developer, I'd like better handling around when the config file is missing.  Right now, if I forget to set up a config.js, the project won't work correctly.  Maybe it would be nice to have some kind of check if the file exists before starting.  Possibly create one from the example or fail completely if even the example is missing?

**Comment by dmasterson** [comment]
Since the config file contains the login info, failing is probably our only option. However, we can still inform the user that they need a config file. We could assume some defaults but without the login token the bot can't function.

**Comment by adigitalmonk** [comment]
Yeah, that's a really good point.  We can throw an exception or something.  I don't know a lot about NodeJS exceptions, so maybe we can look at a useful way to implement them.

**Comment by adigitalmonk** [comment]
I've resolved this issue with the implementation of the config manager.  Now if you try to start up and the file either doesn't load or has an issue, it'll stop execution and warn the user to run `node setup`
